ouroboros-consensus/src/Ouroboros/Consensus/BlockchainTime/WallClock/Types.hs

```diff
@@ -26,49 +25,16 @@ module Ouroboros.Consensus.BlockchainTime.WallClock.Types (
   , SlotLength
   ) where
 
```

Comment:
Most of this module's content was recently upstreamed to https://github.com/input-output-hk/cardano-base/blob/master/slotting/src/Cardano/Slotting/Time.hs